tsconfig.json

```diff
@@ -1,8 +1,21 @@
 {
-  "include": ["src"],
   "compilerOptions": {
-    "declaration": true,
+    "esModuleInterop": true,
+    "noUnusedLocals": true,
+    "rootDir": "./src",
+    "baseUrl": ".",
+    "outDir": "dist/cjs",
+    "lib": ["dom", "esnext"],
+    "jsx": "react",
+    "declaration": false,
+    "module": "es2015",
+    "moduleResolution": "node",
+    "target": "esnext",
+    "strict": false, // temp since this causes errors on used events.
```

Comment:
Let's fix this issue pre-merge

Reply:
I've done a temporary cast. There's an issue about this here -https://github.com/Microsoft/TypeScript/issues/28357